### PR TITLE
add unit test for the register.

### LINKED ISSUE #189: Add test suit for the register/deploy services.
- [ ] Add test suits for APIs.
- [ ] Add tests for deployment.
- [ ] Add unit tests for the register.
- [ ] Add unit tests for the deployment.

### PR DESCRIPTION
**Pull request by jinyangyang222**
Fixes #189

### CODE REVIEW

File: modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/XpanseTestApplication.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * Main entry class to xpanse runtime. This class can be directly executed to start the server.
+ */
+@SpringBootApplication
+@EnableAsync(proxyTargetClass = true)
+@EnableJpaRepositories(basePackages = "org.eclipse.xpanse")
+@EntityScan(basePackages = "org.eclipse.xpanse")
+@ComponentScan(basePackages = "org.eclipse.xpanse")
+@EnableJpaAuditing
+public class XpanseTestApplication {
```
Comment (swaroopar) [inline modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/XpanseTestApplication.java:26]: is it not possible to simply start the actual main class here with test profile?

Comment (jinyangyang222) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/XpanseTestApplication.java:26]: The actual main class cannot be started here, possible cyclic dependency.

Comment (swaroopar) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/XpanseTestApplication.java:26]: oh ok. yes you are right. then can we move this test directly under runtime package?
If not, i would avoid using this and instead only use other ways to instantiate classes which are needed.

Comment (jinyangyang222) [reply modules/orchestrator/src/test/java/org/eclipse/xpanse/orchestrator/XpanseTestApplication.java:26]: yes ok,move to the runtime package is good, i will do this.

**Review by swaroopar — APPROVED** [review]
looks good.

**Comment by jinyangyang222** [comment]
> @jinyangyang222 is this PR now ready for merge? I see the status has been changed to Draft.

Don't merge now,In the runtime package, I think the testing of API modules is more reasonable,how do you like it?

**Pull request closed without merging**